tests/acceptance/01_vars/04_containers/multiple_cmerge_variable_dynamic.cf

```diff
@@ -0,0 +1,144 @@
+# This policy is synthetic, but represents a real life pattern where data is
+# published from multiple bundles and then merged before use.
+# Note: This has a nearly identical test which uses less variables.
```

Comment:
Nearly identical to which test? It's not evident outside the context of this pull request. If the diff is small, maybe you can even include it in the comments so that the reader knows immediately what you are testing.